.ci/run_docker_with_component_tests.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+set -xe
+
+thisScriptDir="$( dirname "${BASH_SOURCE[0]}" )"
+thisScriptDir="$( realpath "${thisScriptDir}" )"
+
+shouldStartExternalServices=false
+
+function shouldPassEnvVarToDocker () {
+    envVarNameToCheck="$1"
+
+    # Pass all the environment variables whose names start with ELASTIC_APM_
+    if [[ ${envVarNameToCheck} == "ELASTIC_APM_"* ]]; then
+        echo "true"
+    fi
+
+    local envVarNamesToPassToDocker=(COMPONENT_TEST_SCRIPT GITHUB_RELEASES_URL PACKAGE TYPE VERSION)
+
+    for envVarNameToPassToDocker in "${envVarNamesToPassToDocker[@]}"
+    do
+        if [ "${envVarNameToCheck}" == "${envVarNameToPassToDocker}" ] ; then
+            echo "true"
```

Comment:
Should this loop ending the function? I don't recall if `return` works for functions, but otherwise, it will print true and then false, if the loop is not ended

```suggestion
            echo "true"
            return
```

Reply:
You are right - I've added missing `return` after each `echo` (except the last one at the last line of the function)